_topics/en/simplicity.md

```diff
@@ -29,21 +46,25 @@ primary_sources:
 ## true" to bold entry
 optech_mentions:
   - title: OP_CHECKTEMPLATEVERIFY
```

Comment:
Simplicity is just a side note here, I don't think it's index worthy.  Suggest dropping this item.